src/main/proto/marathon.proto

```diff
@@ -141,6 +141,22 @@ message MarathonTask {
     repeated string local_volume_ids = 1;
     required State state = 2;
   }
+  enum MarathonTaskStatus {
```

Comment:
or maybe link to src/main/scala/mesosphere/marathon/core/task/state/MarathonTaskStatus.scala and put the docs there?